ui/raidboss/data/07-dt/raid/r7s.ts

```diff
@@ -1,14 +1,421 @@
+        return output.avoidFlare!();
+      },
+      outputStrings: {
+        avoidFlare: {
+          en: 'Away from Flare',
+        },
+        flare: {
+          en: 'Flare on YOU, Away from party',
+        },
+      },
+    },
+    {
+      // different strats have different players taking these tethers,
+      // so we use a generic callout for now
+      id: 'R7S Demolition Deathmatch',
+      type: 'StartsUsing',
+      netRegex: { id: 'A596', source: 'Brute Abombinator', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get tethers',
+        },
+      },
+    },
+    {
+      id: 'R7S Strange Seeds',
+      type: 'StartsUsing',
+      netRegex: { id: 'A598', source: 'Brute Abombinator', capture: true },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Drop seed',
+        },
+      },
+    },
+    {
+      id: 'R7S Tendrils of Terror',
+      type: 'StartsUsing',
+      netRegex: {
+        id: ['A599', 'A59A', 'A59C', 'A59D'],
+        source: 'Brute Abombinator',
+        capture: false,
+      },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Avoid line AoEs',
+        },
+      },
+    },
+    {
+      id: 'R7S Killer Seeds',
+      type: 'StartsUsing',
+      netRegex: { id: 'A59B', source: 'Brute Abombinator', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: Outputs.stackPartner,
+      },
+    },
+    {
+      id: 'R7S Powerslam',
+      type: 'StartsUsing',
+      netRegex: { id: 'A59E', source: 'Brute Abombinator', capture: false },
+      response: Responses.bigAoe('alert'),
+    },
+    {
+      id: 'R7S Stoneringer 2: Stoneringers',
+      type: 'StartsUsing',
+      netRegex: { id: ['A5A0', 'A5A1'], source: 'Brute Abombinator', capture: true },
+      durationSeconds: (_data, matches) => parseFloat(matches.castTime) + 10,
+      infoText: (data, matches, output) => {
+        data.storedStoneringer = matches.id === 'A5A0' ? 'out' : 'in';
+        data.stoneringer2Followup = true;
+        return output[data.storedStoneringer]!();
+      },
+      outputStrings: {
+        in: {
+          en: 'In (for later)',
+        },
+        out: {
+          en: 'Out (for later)',
+        },
+      },
+    },
+    {
+      id: 'R7S Lashing Lariat',
+      type: 'StartsUsing',
+      netRegex: { id: ['A5A8', 'A5AA'], source: 'Brute Abombinator', capture: true },
+      alertText: (_data, matches, output) =>
+        matches.id === 'A5A8' ? output.right!() : output.left!(),
+      outputStrings: {
+        left: {
+          en: '<== Get Left',
+        },
+        right: {
+          en: 'Get Right ==>',
+        },
+      },
+    },
```

Comment:
This one might benefit from being called out earlier, the 3.7s cast time makes movement impossible for some jobs if they don't pre-position at least at the middle of the boss. The earliest this can be determined is when the boss starts casting the preceeding `Brutish Swing`. The boss has turned to face the target wall, and the weapon indicating the wall mechanic and lariat side is glowing. So if boss is facing north, and left (west) weapon is glowing, the lariat will be on the east after boss jumps to wall and spins.

Reply:
I think that is an improvement that can be added later.

Reply:
@valarnin I added an earlier Lashing Lariat call, but I'm not sure if it overlaps too much with the Brutish Swing call.  Let me know what you think.